docs/_posts/2021-11-29-luau-recap-november-2021.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Luau Recap: November 2021"
+---
+
+Luau is our new language that you can read more about at [https://luau-lang.org](https://luau-lang.org).
+
+[Cross-posted to the [Roblox Developer Forum](https://devforum.roblox.com/t/luau-recap-november-2021/).]
+
+## Type packs in type aliases
+
+Type packs are the construct Luau uses to represent a sequence of types. We've had syntax for generic type packs for a while now, and it sees use in generic functions, but it hasn't been available in type aliases. That has changed, and it is now syntactically legal to write the following type alias:
+```lua
+type X<A...> = () -> A...
+type Y = X<number, string>
+```
+
+We've also added support for explicit type packs. Previously, it was impossible to instantiate a generic with two or more type pack parameters, because it wasn't clear where the first pack ended and the second one began. We have introduced a new syntax for this use case:
+```
+type Fn<P..., R...> = (P...) -> R...
+type X = Fn<(number, string), (string, number)>
+```
+
+For more information, check out [the RFC for this feature](https://github.com/Roblox/luau/blob/f86d4c6995418e489a55be0100159009492778ff/rfcs/syntax-type-alias-type-packs.md).
```

Comment:
New documentation section can be linked as well https://luau-lang.org/typecheck#type-packs